lib/dbservice/users.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Dbservice.Users do
+  @moduledoc """
+  The Users context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Dbservice.Repo
+
+  alias Dbservice.Users.User
+
+  @doc """
+  Returns the list of user.
+
+  ## Examples
+
+      iex> list_user()
+      [%User{}, ...]
+
+  """
+  def list_user do
+    Repo.all(User)
```

Comment:
This function is used from `index` function in the controller. However, that endpoint is disabled as I've only added create, update and show endpoint in `router.ex` file. All other endpoints, including index, will return 404 response in here. Maybe when we need this, we can look into putting some pagination protocol.

Reply:
I can't think of any service that has a method to get all users. But if endpoint is disabled, then no problem. It can behere.

Maybe rename to `get_all_users`?